### PR TITLE
[ELEMENTS_SCREENS] [SWE-1815, 1816, 1817] Created InputScreen, ButtonScreen, AvatarScreen #done

### PR DESCRIPTION
**Pull request by Anhelina4**
image
image
image

### CODE REVIEW

File: src/app/screens/Elements/ElementsContent/InputScreen/InputScreen.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import { ContentScreen } from '~/app/components'
+import { Input } from '~/app/components'
+import { InputList } from '~/app/domains/Input/components'
+import { PositionedContainer } from '~/app/components'
```
Comment (Olexandr25) [inline src/app/screens/Elements/ElementsContent/InputScreen/InputScreen.js:4]: rewrite please to this
import { 3 components } from '~/app/components'
import {1 component } from '~/app/domains'

File: src/app/components/elements/Avatar/Avatar.js [hunk 2]
```diff
@@ -16,14 +16,22 @@ import PropTypes from 'prop-types'
  * @returns The AvatarStyled component with the size, state, and tabIndex props.
```
Comment (Olexandr25) [inline src/app/components/elements/Avatar/Avatar.js:16]: I don't see all props in JS doc

File: src/app/components/elements/Input/Input.js [hunk 3]
```diff
@@ -14,7 +14,7 @@ import React from 'react'
  * @returns A React component - Input
```
Comment (Olexandr25) [inline src/app/components/elements/Input/Input.js:14]: add all props to js doc

File: src/app/domains/Avatar/components/AvatarList/AvatarList.js [hunk 4]
```diff
@@ -5,7 +5,7 @@ import {
 } from '../../__constants__'
 import { Col, List, Row, Title } from '~/app/components'
 
-import { AvatarListStyled } from './AvatarListStyled.styled'
+import AvatarListStyled from './AvatarListStyled.styled'
 import { AvatarSimpleView } from '../../components'
```
Comment (Olexandr25) [inline src/app/domains/Avatar/components/AvatarList/AvatarList.js:9]: rewrite to '~app/components'

File: src/app/domains/Button/components/ButtonList/ButtonList.js [hunk 5]
```diff
@@ -1,4 +1,4 @@
-import { Col, List, Row } from '~/app/components'
+import { Box, List } from '~/app/components'
 
 import { BUTTONS_VARIANTS } from '../../__constants__'
 import { ButtonSimpleView } from '../../components'
```
Comment (Olexandr25) [inline src/app/domains/Button/components/ButtonList/ButtonList.js:4]: rewrite path to files

File: src/app/domains/Button/components/ButtonList/ButtonList.js [hunk 6]
```diff
@@ -1,4 +1,4 @@
-import { Col, List, Row } from '~/app/components'
+import { Box, List } from '~/app/components'
```
Comment (Olexandr25) [inline src/app/domains/Button/components/ButtonList/ButtonList.js:1]: you can use this path

File: src/app/domains/Input/components/InputList/InputList.js [hunk 7]
```diff
@@ -1,5 +1,4 @@
-import { Col, Row } from '~/app/components'
-
+import { Box } from '~/app/components'
 import { INPUT_VARIANTS } from '../../__constants__'
 import { InputSimpleView } from '../../components'
```
Comment (Olexandr25) [inline src/app/domains/Input/components/InputList/InputList.js:3]: rewrite path

File: src/app/screens/Elements/ElementsContent/AvatarScreen/AvatarScreen.js [hunk 8]
```diff
@@ -0,0 +1,28 @@
+import { Avatar } from '~/app/components'
+import { AvatarList } from '~/app/domains/Avatar/components'
+import { ContentScreen } from '~/app/components'
+import { PositionedContainer } from '~/app/components'
```
Comment (Olexandr25) [inline src/app/screens/Elements/ElementsContent/AvatarScreen/AvatarScreen.js:4]: rewrite please to this
import { 3 components } from '~/app/components'
import {1 component } from '~/app/domains'

File: src/app/screens/Elements/ElementsContent/ButtonScreen/ButtonScreen.js [hunk 9]
```diff
@@ -0,0 +1,30 @@
+import { Button } from '~/app/components'
+import { ButtonList } from '~/app/domains/Button/components'
+import { ContentScreen } from '~/app/components'
+import { PositionedContainer } from '~/app/components'
```
Comment (Olexandr25) [inline src/app/screens/Elements/ElementsContent/ButtonScreen/ButtonScreen.js:4]: rewrite please to this
import { 3 components } from '~/app/components'
import {1 component } from '~/app/domains'

File: src/app/screens/Elements/index.js [hunk 10]
```diff
@@ -1,3 +1,10 @@
+import { AvatarScreen, ButtonScreen, InputScreen } from './ElementsContent'
+
 import { default as ElementsIntroScreen } from './ElementsIntro'
```
Comment (Olexandr25) [inline src/app/screens/Elements/index.js:3]: strange import, not like usual
try to rewrite this import

File: src/app/components/bundles/ContentScreen/ContentScreen.js [hunk 11]
```diff
@@ -5,7 +5,7 @@ import {
   Content,
   PageContainer,
   ScreenContainer
-} from 'components/bundles/ContentScreen/components'
+} from '~/app/components/bundles/ContentScreen/components'
```
Comment (alexpassss) [inline src/app/components/bundles/ContentScreen/ContentScreen.js:8]: strange path ~

File: src/app/domains/Avatar/components/AvatarList/AvatarList.js (hunk 4 above)
Comment (alexpassss) [reply src/app/domains/Avatar/components/AvatarList/AvatarList.js:9]: ~ ? bad path

should be from 'components'

File: src/app/screens/Elements/ElementsContent/InputScreen/InputScreen.js (hunk 1 above)
Comment (Anhelina4) [reply src/app/screens/Elements/ElementsContent/InputScreen/InputScreen.js:4]: Oh, thanks

**Review by Olexandr25 — APPROVED** [review]

File: src/app/components/bundles/ContentScreen/ContentScreen.js [hunk 12]
```diff
@@ -5,7 +5,7 @@ import {
...
 import { Footer, Header } from 'components'
```
Comment (ArtemSuhoy) [inline src/app/components/bundles/ContentScreen/ContentScreen.js:9]: Rewrite to ~/app/

File: src/app/screens/Logo/index.js [hunk 13]
```diff
@@ -5,7 +5,7 @@ import {
   LogoWordmarkScreen
 } from './LogoContent'
 
-import { default as LogoIntroScreen } from './LogoIntro'
+import LogoIntroScreen from './LogoIntro/LogoIntroScreen'
```
Comment (ArtemSuhoy) [inline src/app/screens/Logo/index.js:8]: Maybe import { LogoIntroScreen } from './LogoIntro ?

File: src/app/screens/Logo/LogoIntro/LogoIntroScreen/LogoIntroScreen.js [hunk 14]
```diff
@@ -1,6 +1,6 @@
 import { IntroScreen } from '~/app/components'
-import { LOGO_LIST } from '~/app/__constants__'
 import React from 'react'
```
Comment (ArtemSuhoy) [inline src/app/screens/Logo/LogoIntro/LogoIntroScreen/LogoIntroScreen.js:2]: Is it necessary?

File: src/app/domains/Avatar/components/AvatarList/AvatarList.js [hunk 15]
```diff
@@ -5,8 +5,8 @@ import {
 } from '../../__constants__'
```
Comment (whoiam460) [inline src/app/domains/Avatar/components/AvatarList/AvatarList.js:5]: please change to import standarts ~/app/

File: src/app/screens/Iconography/IconographyIntro/IconographyIntroScreen/IconographyIntroScreen.js [hunk 16]
```diff
@@ -1,6 +1,6 @@
-import { ICO_LIST } from '~/app/__constants__'
 import { IntroScreen } from '~/app/components'
 import React from 'react'
```
Comment (ArtemSuhoy) [inline src/app/screens/Iconography/IconographyIntro/IconographyIntroScreen/IconographyIntroScreen.js:2]: Remove all useless import React

File: src/app/screens/Color/ColorIntro/ColorIntroScreen/ColorIntroScreen.js [hunk 17]
```diff
@@ -1,6 +1,6 @@
-import { COLOR_LIST } from '~/app/__constants__'
 import { IntroScreen } from '~/app/components'
 import React from 'react'
```
Comment (ArtemSuhoy) [inline src/app/screens/Color/ColorIntro/ColorIntroScreen/ColorIntroScreen.js:2]: same

File: src/app/components/elements/Avatar/Avatar.js [hunk 18]
```diff
@@ -39,7 +52,9 @@ Avatar.propTypes = {
   icon: PropTypes.string,
   imageSrc: PropTypes.string,
   iconSize: PropTypes.string,
-  userInitials: PropTypes.string
+  iconColor: PropTypes.string,
+  userInitials: PropTypes.string,
+  scale: PropTypes.string
 }
```
Comment (ArtemSuhoy) [inline src/app/components/elements/Avatar/Avatar.js:59]: Not enought state and size

File: src/app/screens/Elements/ElementsContent/ButtonScreen/ButtonScreen.js [hunk 19]
```diff
@@ -0,0 +1,29 @@
...
+  return (
+    <ContentScreen
+      key="ButtonScreen"
+      chapter="Button"
+      domainName="Elem."
+      pageNumber="29"
+      breadcrumbs={['Brand', 'Guidelines']}
+      asideTopContent={<ButtonList />}
+      content={
+        <PositionedContainer bottom="-80px" right="33%">
```
Comment (whoiam460) [inline src/app/screens/Elements/ElementsContent/ButtonScreen/ButtonScreen.js:19]: maybe it better to use just px or %

File: src/app/screens/Logo/index.js (hunk 13 above)
Comment (Anhelina4) [reply src/app/screens/Logo/index.js:8]: There was problems with web-pack, I wrote about it, id didn`t work with this import

File: src/app/screens/Logo/LogoIntro/LogoIntroScreen/LogoIntroScreen.js (hunk 14 above)
Comment (Anhelina4) [reply src/app/screens/Logo/LogoIntro/LogoIntroScreen/LogoIntroScreen.js:2]: Yes, it is needed

**Review by ArtemSuhoy — APPROVED** [review]

**Review by whoiam460 — APPROVED** [review]

**Pull request merged**